terraform/azure.tf

```diff
@@ -386,39 +386,39 @@ resource "null_resource" "set_log_profile_retention" {
 # Use only when testing MSI access controls
 resource "azurerm_public_ip" "public_ip" {
   name                         = "Inspec-PublicIP-1"
-  count                        = "${var.public_vm_count}"
-  location                     = "${var.location}"
-  resource_group_name          = "${azurerm_resource_group.rg.name}"
+  count                        = var.public_vm_count
+  location                     = var.location
+  resource_group_name          = azurerm_resource_group.rg.name
   public_ip_address_allocation = "dynamic"
 }
 
 resource "azurerm_network_interface" "nic2" {
   name                = "Inspec-NIC-2"
-  count               = "${var.public_vm_count}"
-  location            = "${var.location}"
-  resource_group_name = "${azurerm_resource_group.rg.name}"
+  count               = var.public_vm_count
+  location            = var.location
+  resource_group_name = azurerm_resource_group.rg.name
 
   ip_configuration {
     name                          = "ipConfiguration1"
-    subnet_id                     = "${azurerm_subnet.subnet.id}"
+    subnet_id                     = azurerm_subnet.subnet.id
     private_ip_address_allocation = "dynamic"
-    public_ip_address_id          = "${azurerm_public_ip.public_ip.0.id}"
+    public_ip_address_id          = azurerm_public_ip.public_ip[0].id
   }
 }
 
 resource "azurerm_virtual_machine" "vm_linux_external" {
   name                  = "Linux-External-VM"
-  count                 = "${var.public_vm_count}"
-  location              = "${var.location}"
-  resource_group_name   = "${azurerm_resource_group.rg.name}"
-  network_interface_ids = ["${azurerm_network_interface.nic2.0.id}"]
+  count                 = var.public_vm_count
+  location              = var.location
+  resource_group_name   = azurerm_resource_group.rg.name
+  network_interface_ids = [azurerm_network_interface.nic2[0].id]
   vm_size               = "Standard_DS2_v2"
 
-  tags {
+  tags = {
     Description = "Externally facing Linux machine with SSH access"
   }
 
-  identity = {
```

Comment:
Should this `=` have been removed? The trend in this PR seems to be adding `=` rather than removing.

Reply:
Its a change in how the provider handles the arguments (block vs hash). Not sure why things have changed but here we are 🤷‍♂ .